.github/workflows/ci.yml

```diff
@@ -38,6 +38,7 @@ jobs:
         env:
           RAILS_ENV: test
           STRIPE_PRIVATE_KEY: ${{ secrets.STRIPE_PRIVATE_KEY }}
+          PADDLE_PUBLIC_KEY_BASE64: MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEA3CGT82ixrOTpCjAyY9pIGthIx4HzZtnO519VECl7wouV4nSD/09YDkyeujygarlYuKjILra4/yuwHoQGD/EnEdADYfNgBNYMuiUhsiXArPQRuqRKSi3xboZEkeDLaeABtxr/5VWyHFtxbSe8FMa6st0Lm6zrOauYkYRbWF+8+6pp4CVNsFSBjP8PLum0zN9Uh44DyFI6qlJ3xA5Uxcr/Ew/Fp3eDPloqqa2MNCZnkJwft5rLVz/B5YWf5jY515OKl+OU/t+FCtxKpjj74ug79Vk5cZsA3044VClsusI3qj/iRXPWtBNKUpj0tknT5Q9J94twv3kfa+XMu61uOw3kVjhf62W6CYt5jtQdk/HgxzMP64ouvf0zVDKHoJtprU524IXOYxK7tiDXuktpVAw3SnNh/bpN0qD2D7TGVzHeajgcppxjBYJc6OzXHToiyIiLGqZdP0ewzfOIKGkW9tOiY/1kSzlkJrQ6sfM5aKg6SSsos0+TBY9t7/UZFVKKe7J4ijSRmsFd+DVmc2KHHReKrd1tEBa2GQeJU9F0iM5EVcYnXeXzHRNha54snsp5ZYXSAmsYZCAutdh+gWI2AlniP97ZBsCbv7RHLQOY60CXkKnILEPhu4u5Kp7P1Ez1deGf/mcnZz21RCdC7PhqPXtmWtpkNrCK6BeRMj0XZG7oB3MCAwEAAQ==
```

Comment:
I am not sure how to handle that. This is a paddle test account public key which is required for the signature verification test. Because it is a public key and not a private one, it does not necessarily have to be secret. 

Because a VCR cassette is used for the signature test, you also need this public key for the test to be successful. That's why I added it to the repository. Does it make more sense to hardcode the webhook object and the public key into the signature test? Otherwise, the public key must be configurable in the event that the vcr cassettes change.

Reply:
Correct, but if we don't all share the same account, this key needs to be changeable right?

Reply:
I rechecked the code and only the fixture json files are used for the signature tests. So it is not necessary to update the public key if the VCR cassettes have changed. 
I have now put the key in a separate file (`support/fixtures/verification/paddle_public_key.pem`). I think this makes sense because you only have to update the key if you are also updating the fixture json files.